### ISSUE TITLE
Error in gridsome develop

### ISSUE THREAD

**Issue by ekoeryanto** [gridsome/gridsome#11]
```
SyntaxError: Invalid regular expression: /^\assets\static\(.*)\/?$/: Unmatched ')'
    at new RegExp (<anonymous>)
    at pathtoRegexp (O:\Repository\gridsome-site\node_modules\express\node_modules\path-to-regexp\index.js:128:10)
    at new Layer (O:\Repository\gridsome-site\node_modules\express\lib\router\layer.js:45:17)
    at Function.route (O:\Repository\gridsome-site\node_modules\express\lib\router\index.js:494:15)
    at Function.app.(anonymous function) [as get] (O:\Repository\gridsome-site\node_modules\express\lib\application.js:481:30)
    at module.exports (O:\Repository\gridsome-site\node_modules\gridsome\lib\server\createExpressServer.js:25:10)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

I believe that this error only happen on windows system, but I'll try to investigate it more.

**Comment by ekoeryanto** [comment]
issue #13 is more global than this.

**Comment by hjvedvik** [comment]
Hi, which Gridsome version are you using? There was an issue on windows in v0.2, but should be fixed in v0.2.2.

**Comment by ekoeryanto** [comment]
yes, it fixed in 0.2.2, thanks
